pkg/istio-agent/agent.go

```diff
@@ -88,6 +88,9 @@ const (
 	MetadataClientCertKey   = "ISTIO_META_TLS_CLIENT_KEY"
 	MetadataClientCertChain = "ISTIO_META_TLS_CLIENT_CERT_CHAIN"
 	MetadataClientRootCert  = "ISTIO_META_TLS_CLIENT_ROOT_CERT"
+	CertSignerLabel         = "cert.istio.io/signer"
+	CertSignerNSLabel       = "namespace:"
```

Comment:
what does the namespace do?

Reply:
This is the prefix:  If   signer is defined via pod label, eg: `cert.isito.io/signer=prod`, the generated signer info will be `pod:prod` , If   namespace name is used as  signer info, eg: `staging`,  the generated signer info will be `namespace:staging` , then we can see which one has higher priority when comparing with the one defined via env  `K8S_SIGNER `